sources/packages/backend/apps/api/src/route-controllers/application/models/application.dto.ts

```diff
@@ -40,6 +46,12 @@ export class SaveApplicationAPIInDTO {
    */
   @IsPositive()
   programYearId: number;
+  /**
+   * Selected offering intensity of the application.
+   */
+  @IsOptional()
```

Comment:
Please create a specific DTO for each operation, or please explain the reason why this would be a better approach.